packages/mui-styled-engine-sc/src/GlobalStyles/GlobalStyles.js

```diff
@@ -19,5 +19,10 @@ export default GlobalStyles;
 
 GlobalStyles.propTypes = {
   defaultTheme: PropTypes.object,
-  styles: PropTypes.oneOfType([PropTypes.string, PropTypes.object, PropTypes.func]),
+  styles: PropTypes.oneOfType([
+    PropTypes.array,
```

Comment:
`array` was missing which breaks the test when using `<GlobalStyles styles={[…]}`.

Both emotion and styled-components support array.